.drone/pipelines/secrets.jsonnet

```diff
@@ -0,0 +1,23 @@
+local newSecret(name) = {
```

Comment:
The `pipelines` folder is for jobs which run in Drone, can this be moved to util instead?

Reply:
Done!